hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -262,12 +264,9 @@
     "hbase.hregion.special.recovered.edits.dir";
 
   /**
-   * Whether to use {@link MetaCellComparator} even if we are not meta region. Used when creating
-   * master local region.
+   * The {@link CellComparator} class we want to use. Used when creating master local region.
    */
-  public static final String USE_META_CELL_COMPARATOR = "hbase.region.use.meta.cell.comparator";
-
-  public static final boolean DEFAULT_USE_META_CELL_COMPARATOR = false;
+  public static final String CELL_COMPARATOR_IMPL = "hbase.region.cell.comparator.impl";
```

Comment:
Why need to expose this config? This should be a internal thing?

Reply:
We need to set it in MasterRegion. And HRegion is IA.Private so no big problem?